pkgs/by-name/ep/epsonscan2/package.nix

```
@@ -152,22 +154,27 @@ stdenv.mkDerivation {
     inherit description;
     mainProgram = "epsonscan2";
     longDescription = ''
-      Epson Scan 2 scanner driver including optional non-free plugins such as OCR and network
-      scanning.
+      The Epson Scan 2 scanner driver, including optional non-free plugins such
+      as OCR and network scanning.
 
       To use the SANE backend:
-      <literal>
-      hardware.sane.extraBackends = [ pkgs.epsonscan2 ];
-      </literal>
-
-      Overrides can be used to customise this package. For example, to enable non-free plugins and
-      disable the Epson GUI:
-      <literal>
-      pkgs.epsonscan2.override { withNonFreePlugins = true; withGui = false; }
-      </literal>
+      ```nix
+      {
+        hardware.sane.extraBackends = [ pkgs.epsonscan2 ];
+      }
+      ```
+
+      Overrides can be used to customise this package. For example, to enable
+      non-free plugins and disable the Epson GUI:
+      ```nix
+      pkgs.epsonscan2.override {
+        withNonFreePlugins = true;
+        withGui = false;
+      }
+      ```
     '';
     homepage = "https://support.epson.net/linux/en/epsonscan2.php";
-    platforms = [ "x86_64-linux" ];
+    platforms = lib.systems.inspect.patternLogicalAnd lib.systems.inspect.patterns.isUnix lib.systems.inspect.patterns.isx86;
```

```suggestion
    platforms = lib.systems.inspect.patternLogicalAnd lib.systems.inspect.patterns.isLinux lib.systems.inspect.patterns.isx86;
```
Package is not relevant for Darwin

---

Fixed. Thanks for catching that!